<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HillshadeWidget</class>
 <widget class="QWidget" name="HillshadeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Algorithm:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="algComboBox">
     <item>
      <property name="text">
       <string>ZevenbergenThorne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Horn</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="lightGroupBox">
     <property name="title">
      <string> Light direction:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="0">
       <widget class="QRadioButton" name="multiDirectionRadioButton">
        <property name="text">
         <string>Multidirectional shading</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Altitude:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="azimuthSpinBox">
           <property name="prefix">
            <string/>
           </property>
           <property name="suffix">
            <string> °</string>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>315.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Azimuth: </string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDoubleSpinBox" name="altitudeSpinBox">
           <property name="prefix">
            <string/>
           </property>
           <property name="suffix">
            <string> °</string>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
           <property name="value">
            <double>45.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QCheckBox" name="combinedCheckBox">
           <property name="text">
            <string>Combine with slope</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="userDefRadioButton">
        <property name="text">
         <string>User defined</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="vscalingGroupBox">
     <property name="title">
      <string>Vertical scaling:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="vexaCheckBox">
        <property name="text">
         <string>Exaggeration</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="vexaSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Elevation unit:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="vunitComboBox">
        <item>
         <property name="text">
          <string>Meters</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Us Feet</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>50</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>userDefRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vexaCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>vexaSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>166</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
